eng/Build.ps1

```diff
@@ -247,7 +247,7 @@ function VerifyAssemblyVersionsAndSymbols() {
     }
 }
 
-function TestUsingNUnit([string] $testProject, [string] $targetFramework) {
+function TestUsingNUnit([string] $testProject, [string] $targetFramework, [boolean] $noTestFilter = $false) {
```

Comment:
Temporary workaround to not pass filter to xunit runner (since it doesn't know "TestCategory" filter, itwill skip all tests in this case).
Long-term solution will be to migrate these tests to xUnit and use TraitAttribute's Category parameter.